macos/flutter_webrtc.podspec

```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'flutter_webrtc'
-  s.version          = '0.7.1'
+  s.version          = '0.8.11'
```

Comment:
that's a large jump. why `0.8.11` instead of `0.8.0`?

Reply:
I want to make its version consistent with flutter-webrtc, but it seems difficult to align the two, so you are right, just jump to 0.8.0.

Reply:
oh got it.. making them aligned makes a lot of sense!